ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ITGuru.FourWheels.Data
+{
+    public class Jasmin
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int Age { get; set; }
```

Comment:
Use uint instead?

Reply:
I might ask stupidly but, why is that smarter than a proper int in this case?

Reply:
I suppose that your age will never be negative. We can use the uint type to only allow positive numbers. Additionally, we can save space if we decide to use ushort instead.

Reply:
Got it. I will change that and make a new PR.